CLA/CLA-v1.0.md

```diff
@@ -1,24 +0,0 @@
-
-1.	You represent that you are legally entitled to grant the above license. If your employer(s) has rights to intellectual property that you create that includes your Contributions, you represent that you have received permission to make Contributions on behalf of that employer, that your employer has waived such rights for your Contributions to Freshli, or that your employer has executed a separate Corporate CLA with Freshli.
-
-1.	You represent that each of Your Contributions is Your original creation (see section 7 for submissions on behalf of others). You represent that Your Contribution submissions include complete details of any third-party license or other restriction (including, but not limited to, related patents and trademarks) of which you are personally aware and which are associated with any part of Your Contributions.
-
-1.	You are not expected to provide support for Your Contributions, except to the extent You desire to provide support. You may provide support for free, for a fee, or not at all. Unless required by applicable law or agreed to in writing, You provide Your Contributions on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied, including, without limitation, any warranties or conditions of TITLE, NON- INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE.
-
-1.	Should You wish to submit work that is not Your original creation, You may submit it to Freshli separately from any Contribution, identifying the complete details of its source and of any license or other restriction (including, but not limited to, related patents, trademarks, and license agreements) of which you are personally aware, and conspicuously marking the work as "Submitted on behalf of a third-party: [named here]".
-
-1.	You agree to notify Freshli of any facts or circumstances of which you become aware that would make these representations inaccurate in any respect.
-
-_Revised: November 23, 2020._
```

Comment:
If we're going to remove the CLA from this repository, then I think we need to add a note in the README to the location where the document can be found along with a mention that it has to be signed when submitting a pull request.

Reply:
Good point! I've added it to `CONTRIBUTING.md`. I think that makes more sense because we only want it signed if someone contributes.